Python/Product/Analysis/LanguageServer/Enums.cs

```diff
@@ -71,9 +71,30 @@ public enum TextDocumentSyncKind : int {
     }
 
     public enum MessageType : int {
+        /// <summary>
+        /// General language server output relevant to the user
+        /// such as information on Python interpreter type.
+        /// </summary>
+        General = 0,
```

Comment:
This value isn't defined in LSP, so should start with an underscore (for consistency with other unspecified values)